pyedflib/tests/test_edfreader.py

```diff
@@ -125,12 +135,12 @@ def test_EdfReader_BDF_datarec_2(self):
         np.testing.assert_equal(f.datarecords_in_file, datarecords / datarecord_duration)
         np.testing.assert_equal(f.getFileDuration(), datarecords)
         
-        sample_frequencies = [1000, 800, 500, 975, 999]
+        sample_frequencies = [500, 400, 250, 487.5, 499.5]
```

Comment:
these were wrong before. Now they are correct and finally are also what EDFBrowser would display.